guides/space_status_history.md

```diff
@@ -0,0 +1,56 @@
+---
+copyright: 'Copyright IBM Corp. 2017'
+link: 'space-status-history'
+is: 'experimental'
+---
+
+# Space Status History
+
+## Concepts
+
+The _statusHistory_ field provides a list containing information about status changes 
+in a space. The most recent value for each space status option is stored, ie. when a value is selected multiple times the old values are overwritten. 
+For a new space initially the statusHistory will be empty. 
+A statusHistory entry will be created when a user selects a new status value in the space.
+
+The statusHistory data is eventually consistent and not synchronized with a status update.
```

Comment:
what does this mean, it's not clear

Reply:
Which part specifically? Just line 16 or the whole description?

Reply:
just line 16

Reply:
This means when you update the status history the response will not contain the update. The update is propagated to the status history store later (usually in <1 second). 

Another area that is eventually consistent like this is system messages, e.g. when you add someone to a space the system message is created a short time later.

Reply:
could you add that qualifier as another sentence here so it's clear?

Reply:
@jarusso and @andrewegan I think much of this will make more sense in the context of also having a "concepts" or "introduction" type doc for the topic of Watson Work template-based Solutions.  Something that we should aim to have within the Q1 goals